### PR TITLE
Migrate mockery to packages feature

### PR DESCRIPTION
**Pull request by KuSh**
#### Summary
Fix mockery warning about config deprecation
```sh
WRN DEPRECATION: use of the packages config will be the only way to generate mocks in v3. Please migrate your config to use the packages feature. dry-run=false migration=https://vektra.github.io/mockery/v2.42/migrating_to_packages/ url=https://vektra.github.io/mockery/v2.42/features/#packages-configuration version=v2.42.2
```

I didn't migrate the enterprise ldap part as I don't have access to that part of the code

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by agnivade** [comment]
Nice on @KuSh

File: server/einterfaces/mocks/ElasticsearchFixChannelIndexInterface.go [hunk 1]
```diff
@@ -1,46 +0,0 @@
-// Code generated by mockery v2.23.2. DO NOT EDIT.
```
Comment (agnivade) [inline server/einterfaces/mocks/ElasticsearchFixChannelIndexInterface.go:1]: These should not be deleted. Can you revert these changes?

**Review by agnivade — CHANGES_REQUESTED** [review]

File: server/einterfaces/mocks/ElasticsearchFixChannelIndexInterface.go (hunk 1 above)
Comment (KuSh) [reply server/einterfaces/mocks/ElasticsearchFixChannelIndexInterface.go:1]: Those files are not generated by the actual or the new mock generation. Therefore I thought those where not needed anymore. I'll revert the commit !

**Comment by KuSh** [comment]
@agnivade I've reverted (apparently unused) mocks deletion!

**Review by agnivade — APPROVED** [review]
Looks good!

**Review by isacikgoz — APPROVED** [review]
LGTM, thanks @KuSh 🚀

**Comment by amyblais** [comment]
@holopin-bot @KuSh Mattermost Most Valued Professional (MVP)
